core/src/test/scala/org/http4s/blaze/pipeline/stages/QuietTimeoutStageSpec.scala

```diff
@@ -2,11 +2,19 @@ package org.http4s.blaze.pipeline
 package stages
 
 import java.nio.ByteBuffer
+import org.http4s.blaze.util.TickWheelExecutor
+import org.specs2.specification.BeforeAfterAll
 import scala.concurrent.duration._
 
-class QuietTimeoutStageSpec extends TimeoutHelpers {
+class QuietTimeoutStageSpec extends TimeoutHelpers with BeforeAfterAll {
 
-  override def genDelayStage(timeout: FiniteDuration): TimeoutStageBase[ByteBuffer] = new QuietTimeoutStage[ByteBuffer](timeout)
+  var scheduler: TickWheelExecutor = null
+
+  override def beforeAll() = { scheduler = new TickWheelExecutor() }
```

Comment:
Better to do this as a per test rig eventually.

Reply:
I can't ever remember how to do this right.

Reply:
Literally I would expose a `Resource[F, TickWheelExecutor]` and then just change tests to `tickWheelExecutor.use(test)`